app/controllers/api/v1/user/words_controller.rb

```diff
@@ -16,6 +16,20 @@ def create
     end
   end
 
+  def show
+    word = Word.find(params[:id])
+    books = Glossary.where('word_id = ?', word.id).pluck(:book_id) #need to also add user_id, passed as a param
+    books.map do |book|
+      # create a method in the bookservice to lookup the volume, grab title and id
+      # BookService.volume_lookup (https://www.googleapis.com/books/v1/volumes/volumeId)
+      # make book POROS
+    end
+
+    # serialize a word object with an attribute that is an array of book objects?
```

Comment:
This for sure, best to send across as compact a passage as possible, it would only need a new serializer if you fed it in correctly. Use a Facade to grab the book information so you can call it here and add it to the word using the serializer when you send it out.